Add documentation about new parameters of the text_line field type (#596)
sulu/sulu-docs: reference/content-types/text_line.rst

    * - headline
      - boolean
      - If true the height and font size of the text line get increased.

>     * - softMaxLength

Author: ```suggestion
    * - soft_max_length
```

      - string
      - Soft limit for maximum number of characters. Will show a character counter (replaces `max_characters`)
    * - max_segments
      - string
      - Soft limit for maximum number of segments. Will show a segment counter.
    * - segment_delimiter
      - string
      - The delimiter used to split the value into segments (required to use `max_segments`)

>     * - minLength
>       - string
>       - The minimum number of characters
>     * - maxLength

Author: ```suggestion
    * - min_length
      - string
      - The minimum number of characters
    * - max_length
```

    * - headline
      - boolean
      - If true the height and font size of the text line get increased.
    * - min_characters
      - string

>       - The minimum amount of characters

Reviewer: would use `number` instead of `amount`

    * - headline
      - boolean
      - If true the height and font size of the text line get increased.
    * - softMaxLength
      - string

>       - Soft limit for maximum amount of characters by showing a character counter (replaces `max_characters`)

Reviewer: ```suggestion
      - Soft limit for maximum number of characters. This will show a character counter (replaces `max_characters`)
```
